docs/changelog.md

```diff
@@ -11,6 +11,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/main/.config.yml)
 
+* **support-utils**
+  * ⚠️ **Add Firefox Focus packages to known browsers list
```

Comment:
nit: Normally when we add something we use the 🌟️ symbol :) 
We use the ⚠️ for breaking changes.